### DISCUSSION TITLE
Repository branch list alpha feedback

### DISCUSSION THREAD

**Discussion by shaedrich** [community/community#70534] [Repositories] [answered]
### Select Topic Area

Product Feedback

### Body

Information-wise, the view looks pretty much the same, except for the cleaner and more state of the art UI.

Since these are tabs now, it would be nice to have a badge counter next to the tab title: ![grafik](https://github.com/community/community/assets/11225821/5519b045-dfc2-4c17-80ce-e8abb98faaec)

While looking through the tabs, however, I found something that looked not quite right: The ["stale" tab](https://github.com/github/explore/branches/stale) lists the branch `gh-pages` twice with identical information except for the "Update" column. ![grafik](https://github.com/community/community/assets/11225821/f69aea66-27bd-4257-85dc-5e2a9722ae71)

~~Apropos: The [feedback link](https://github.com/github/repos/discussions/8863) is broken. It navigates to the repository `github/repos` which is either non-existent or private (in the latter case just ignore my feedback) ![grafik](https://github.com/community/community/assets/11225821/14ff9c5e-fd88-4a5c-86c2-0315a09a0424)~~

While you are at it, do you consider adding this functionality: #69478?

**Comment by patrick-knight** [comment, marked as answer]
Thanks for the 👀 _early_ feedback @shaedrich. This snuck out a bit early in a few places, keep your eye out for a wider feature preview coming soon not with broken links. 

The badge counts are an interesting idea, but we'll have to think about the performance impact on large repos.

A fix of double counting is a work in progress right now. 

As far as #69478 this is really interesting feedback we had not considered yet. It could be hard to pull off well for repo that have a lot of forks and open PRs but I'm going to chat about it with the team.

**Reply by shaedrich** [reply]
Ah, that's why ^^ Thanks for the explanation @patrick-knight. Sounds interesting 👍🏻 

Especially "stale" could be something that would be cached&mdash;but that depends on your implementation

As for #69478 on the other hand, the good thing is that in this case, mostly forks open PRs towards the parent, so usually, they aren't forked too much in a nested manner as far as I know. If it was the other way round, sure, that would be a lot data.

**Reply by patrick-knight** [reply]
Here's the official [feature preview](https://github.blog/changelog/2023-10-23-new-branches-and-commits-pages-feature-preview/) announcement.

**Reply by shaedrich** [reply]
Thanks a lot for the heads-up! I'll check it out.
